cms/envs/aws.py

```diff
@@ -373,3 +373,14 @@
 
     # third_party_auth config moved to ConfigurationModels. This is for data migration only:
     THIRD_PARTY_AUTH_OLD_CONFIG = AUTH_TOKENS.get('THIRD_PARTY_AUTH', None)
+
+########## django-sudo ##########
+if FEATURES.get('ENABLE_DJANGO_SUDO', False):
+    # force re-authentication before activating administrative functions
+    MIDDLEWARE_CLASSES += ('sudo.middleware.SudoMiddleware',)
+
+    # Allows sudo-mode
+    INSTALLED_APPS += (
+        'sudo',
+        'django_sudo_helpers'
+    )
```

Comment:
If this is only in `aws.py`, then it won't work in other configurations (like the non-devstack install that I use locally). In the past, the way we've handled this was to define a function in `common.py` that made the changes, and then called it at the end of the commonly used environment files. It would be nice to have a better pattern for post-aws configuration changes, though.